app/views/content/is-teaching-right-for-me/qualifications-you-need-to-teach.md

```diff
@@ -54,12 +54,14 @@ This could be through having a degree in the subject, or having:
 * a degree related to but not in the subject – for example, your degree is in engineering but you’d like to teach physics
 * an unrelated degree but relevant professional experience
 
+You can also train to teach primary with a specialist subject, for example primary with maths, so you can develop your knowledge of that particular subject.
+
+Or you can train to teach two subjects at secondary, for example PE with computing, so you can develop the skills to teach in an additional subject.
```

Comment:
```suggestion
Or you can train to teach two subjects at secondary, for example PE with computing, so you can develop the skills to teach an additional subject.
```

Reply:
Is PE with computing a common combination? If not, is it worth putting one that is more common?

Reply:
@gemmadallmandfe I'm not sure. When I looked on Find it lists courses as PE with an Ebacc subject in the what subject do you want to teach. So I assumed PE with one of these would be the most popular and chose computing so it wasn't the same as the primary example. I can check to see if we have data on which is the most common combination with PE. Or we can say PE with an Ebacc subject and then list out what these are?